### ISSUE TITLE
Circular import issues when setting a custom permissions handler

### ISSUE THREAD

**Issue by jthemphill** [hovel/pybbm#69]
# tl;dr
- To set `PYBB_PERMISSION_HANDLER`, I have to import a `CustomPermissionHandler`.
- To define `CustomPermissionHandler` with proper inheritance, I have to import `DefaultPermissionHandler`
- When I import `DefaultPermissionHandler`, PyBBM checks for  `PYBB_PERMISSION_HANDLER` in the settings---while I'm still importing what I need to even define it!
# Long Version

I'm trying to set a custom permission handler, so I override the settings variable:

```
import myapp.forum.permissions
# ...
PYBB_PERMISSION_HANDLER = myapp.forum.permissions.CustomPermissionHandler
```

All well and good so far, but in order to point to this class I have to actually import its module.

Inside of the imported module, I'm supposed to override `DefaultPermissionHandler`, so I do so:

```
import pybb.permissions

class CustomPermissionHandler(DefaultPermissionHandler):
    """There's some code here."""
```

But now I've imported `pybb.permissions`! That means the code in `pybb.permissions` will run before my settings module has finished being defined. And what's in `pybb.permissions`?

```
perms = _resolve_class(defaults.PYBB_PERMISSION_HANDLER)
```

...when this line of code is executed, pybbm looks for the variable in my settings module. But the interpreter hasn't finished defining my settings yet, so it goes ahead and uses the default permission handler anyway, rendering all my hard work for naught.
# Proposed Solution

Instead of making me import, define, and dig up the class while settings are being defined, let me use a string which resolves to the class's path. Django does this when defining `INSTALLED_APPS`, so there's precedent.
